ui/packages/catalog-ui-search/src/main/webapp/component/tabs/tabs.less

```diff
@@ -1,6 +1,8 @@
 @{customElementNamespace}tabs {
   .customElement;
   background: inherit;
+  border-right: 1px solid fade(contrast(@backgroundAccentContent), 10%);
```

Comment:
It feels like this should be on a different component.  What is the goal of putting a border right on this component?

Reply:
It's intended to be a border that cleanly separates content-left from content-right so that they are two distinct sections (especially if the background colors are the same). It's possible there's a better location for it and I couldn't find it.

Reply:
👍 Okay, we should move it to a different component since this will affect all tab components.

Reply:
Done!